src/js/components/Meter/Meter.js

```diff
@@ -25,18 +27,26 @@ const Meter = forwardRef(
       thickness = 'medium',
       type = 'bar',
       value,
+      reverse,
```

Comment:
```suggestion
      reverse: reverseProp,
```

Reply:
Some small naming/code convention suggestions to align with how we handle in other places.

Reply:
Can we also move the placement of where this is destructured to be in alphabetical order?